provisioning/service/src/Config/SymmetricKeyAttestation.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service
+{
+    /// <summary>
+    /// Attestation using a symmetric key
+    /// </summary>
+    public sealed class SymmetricKeyAttestation : Attestation
+    {
+        /// <summary>
+        /// Default json constructor
+        /// </summary>
+        /// <param name="primaryKey">The primary key to use for attestation</param>
+        /// <param name="secondaryKey">The secondary key to use for attestation</param>
+        [JsonConstructor]
+        public SymmetricKeyAttestation(string primaryKey, string secondaryKey)
+        {
+            try
+            {
+                PrimaryKey = primaryKey;
+                SecondaryKey = secondaryKey;
+            }
+            catch (ArgumentException e)
+            {
+                throw new ProvisioningServiceClientException(e);
+            }
```

Comment:
do we validate anything here???

Reply:
It looks like we do not, so I'll remove the catch statement here